common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2520,4 +2521,13 @@ object CelebornConf extends Logging {
       .doc("The time before a cache item is removed.")
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefaultString("15s")
+
+  val FETCH_FAIL_TRIGGER: ConfigEntry[Int] =
+    buildConf("celeborn.test.client.fetch.fetchFailTrigger")
```

Comment:
The name is confusing, suggest 
- "celeborn.test.client.fetchFailedChuckIndex" or
- "celeborn.test.client.fetch.failedChuckIndex"